src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const res = [];
+  const obj = Object.assign({}, state);
```

Comment:
The naming for the object copy could be more descriptive. Instead of `obj`, a name like `stateCopy` would better convey the purpose of this variable.